docs/docs/providers/osu.md

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_label: osu!
```

Comment:
An entry needs to be added to `docs/sidebar.js` too. Cf. this example -> https://github.com/NangoHQ/nango/pull/540/files

Reply:
oops, done